spanner/protoutils.go

```diff
@@ -128,3 +130,20 @@ func structType(fields ...*sppb.StructType_Field) *sppb.Type {
 func nullProto() *proto3.Value {
 	return &proto3.Value{Kind: &proto3.Value_NullValue{NullValue: proto3.NullValue_NULL_VALUE}}
 }
+
+func protoType() *sppb.Type {
+	return &sppb.Type{Code: sppb.TypeCode_PROTO}
+}
+
+func enumType() *sppb.Type {
+	return &sppb.Type{Code: sppb.TypeCode_ENUM}
+}
+
+func messageProto(m proto.Message) *proto3.Value {
```

Comment:
We are going to support only proto2 for Verily Private GA. Not sure if there is any implication to use proto3.Value here instead of proto2.Value. Can you verify?